From: Federico Ceratto <federico@debian.org>
Date: Tue, 3 Jan 2017 21:30:13 +0000
Updated: Sun, 27 Dec 2020 20:42:08 +0000
Subject: Use packaged decorator.py
Forwarded: not-needed

---
Index: python3-prometheus-client/prometheus_client/context_managers.py
===================================================================
--- python3-prometheus-client.orig/prometheus_client/context_managers.py
+++ python3-prometheus-client/prometheus_client/context_managers.py
@@ -5,7 +5,7 @@ from typing import (
     Union,
 )
 
-from .decorator import decorate
+from decorator import decorate
 
 if TYPE_CHECKING:
     from . import Counter
Index: python3-prometheus-client/tests/test_core.py
===================================================================
--- python3-prometheus-client.orig/tests/test_core.py
+++ python3-prometheus-client/tests/test_core.py
@@ -12,9 +12,16 @@ from prometheus_client.core import (
     HistogramMetricFamily, Info, InfoMetricFamily, Metric, Sample,
     StateSetMetricFamily, Summary, SummaryMetricFamily, UntypedMetricFamily,
 )
-from prometheus_client.decorator import getargspec
 from prometheus_client.metrics import _get_use_created
 
+from inspect import getfullargspec
+
+
+def getargspec(f):
+    """A replacement for inspect.getargspec"""
+    spec = getfullargspec(f)
+    return spec.args, spec.varargs, spec.varkw, spec.defaults
+
 
 def assert_not_observable(fn, *args, **kwargs):
     """
