Index: python3-prometheus-client/pyproject.toml
===================================================================
--- python3-prometheus-client.orig/pyproject.toml
+++ python3-prometheus-client/pyproject.toml
@@ -7,11 +7,7 @@ name = "prometheus_client"
 version = "0.24.1"
 description = "Python client for the Prometheus monitoring system."
 readme = "README.md"
-license = "Apache-2.0 AND BSD-2-Clause"
-license-files = [
-	"LICENSE",
-	"NOTICE",
-]
+license = { file = "LICENSE" }
 requires-python = ">=3.9"
 authors = [
     { name = "The Prometheus Authors", email = "prometheus-developers@googlegroups.com" },
@@ -57,3 +53,17 @@ Documentation = "https://prometheus.gith
 
 [tool.setuptools.package-data]
 prometheus_client = ['py.typed']
+
+[tool.setuptools.data-files]
+"tests" = [
+  "tests/certs/*",
+  "tests/proc/stat",
+  "tests/proc/584/*",
+  "tests/proc/26231/limits",
+  "tests/proc/26231/stat",
+  "tests/proc/26231/fd/*",
+]
+
+[tool.setuptools.packages.find]
+where = ["."]
+exclude = ["debian"]
